src/hyp3_sdk/jobs.py

```diff
@@ -291,3 +291,6 @@ def filter_jobs(
                 filtered_jobs.append(job)
 
         return Batch(filtered_jobs)
+
+    def total_credit_cost(self):
+        return sum(job.credit_cost for job in self.jobs)
```

Comment:
`job.credit_cost` defaults to `None` for jobs that predate the introduction of that field. That results in a `TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'` exception for batches that include older jobs.

```suggestion
        return sum(job.credit_cost for job in self.jobs if job.credit_cost is not None)
```